examples/query_elasticsearch/elasticsearch_queuelength.py

```diff
@@ -0,0 +1,164 @@
+###################################################################
+# Script Name       : elasticsearch_queuelength.py
+#
+# Description       : Queries elasticsearch and calculates the current
+#                     number of citizens in line for a given Service BC
+#                     field office
+#
+#                     Configuration file should contain line separated list of
+#                     Service BC offices to query from Elasticsearch, index and
+#                     endpoint specified.
+#
+#                     In order to query by office location (eg: Kelowna,
+#                     Kamloops), it is necessary to include a JSON export
+#                     of office names and id's. This json export can be
+#                     created in the looker explore, here:
+#                     https://analytics.gov.bc.ca/
+#                       explore/cfms_poc/cfms_poc?toggle=fil&qid=DWe1cjJI5lYB084l8fjZ1M
+#
+#                     Name the json export into a file called
+#                     serviceBCOfficeList.json and place it in the same
+#                     directory as this script.
+#
+# Requirements      : Install the following python packages to allow
```

Comment:
You should add that the `serviceBCOfficeList.json` is a requirement for this script.

Reply:
Added requirement.